st/initscripts

```diff
@@ -18,7 +18,8 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-set -e
+#set -x
+#set -e
```

Comment:
"#set -e" Why it is disabled ? 
Does test complete successfully, if we enable this option ?

Reply:
No, the test does not compile successfully if we enable the option. dix_pver_build() in build_conf() which is called from within motr_service_start() returns a non-zero value, and that is interpreted as an error by the -e option and the conf.xc file is not generated, so the motr services are not able to start due to that.

Reply:
okay.